pyEpiabm/pyEpiabm/sweep/assign_household_ages_sweep.py

```diff
@@ -0,0 +1,394 @@
+#
+# Sweep to assign ages to people in households
+#
+
+import random
+from scipy.stats import poisson
+import numpy as np
+
+from pyEpiabm.core import Person
+from pyEpiabm.core import Parameters
+from .abstract_sweep import AbstractSweep
+
+
+class AssignHouseholdAgesSweep(AbstractSweep):
+    """Class to assign ages to people in
+    households using the same age distribution
+    model as covid-sim.
+    """
+
+    def __init__(self):
+        """Call in variables from the parameters file.
+        """
+        self.age_params = Parameters.instance().household_age_params
+        self.age_proportions = Parameters.instance().age_proportions
+        self.num_age_groups = len(self.age_proportions)
+        self.age_group_width = 5
+
+    def one_person_household_age(self, person: Person):
+        """Method that assigns an age to the person
+        in a one person household. A random number is first drawn
+        that decides which set of conditions the person's age
+        should satisfy, e.g. if they are old or young and single etc. The
+        person's age is then repeatedly set until it makes realsitic sense
+        e.g they are not a child living alone.
+
+        Parameters
+        ----------
+
+        Person : Person
+            Instance of Person class
+
+        """
+
+        r = random.random()
+
+        # case where an elderly person lives alone
```

Comment:
It would be great to have some more documentation on the checks in each loop - i.e. what the break ratio is here etc. If its too much to put in this file, then again the wiki is probably the best place for it, but link it here. I think this would really help our readability over covidsim though

Reply:
To be done